db/seeds.rb

```diff
@@ -3,7 +3,8 @@
 
 User.delete_all
 
-user = User.create(email: "email@email.com", password: "password")
+user = User.create(email: "email@email.com", password: "****",
+  password_confirmation: "****" )
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Space inside parentheses detected.